README.md

```diff
@@ -146,6 +146,10 @@ For a debug build:
 Test
 ----
 
+If a ruby was found during configuration, execute the following command before running tests:
```

Comment:
yeah, it's probably not worth any real effort to screw around with.

We might want to consider moving to the `make check` pattern that lets us add other steps/deps before the test target (this is my preferred solution longer-term), but probably not worth it for this PR